client/src/modules/user/model/actions.js

```diff
@@ -1,13 +1,19 @@
-import { getData, postData, postRequest } from 'common/utils/fetchMethods';
+import {
+  getData,
+  getJson,
+  postData,
+  postRequest
+} from 'common/utils/fetchMethods';
 import { MessageType as NotificationType } from 'common/constants/enums';
 import { createNotificationWithTimeout } from 'modules/notification/model/actions';
 import { ValidationException } from 'common/exceptions/ValidationException';
 import history from 'common/utils/history';
 
 export const AuthorizationActionTypes = Object.freeze({
-  LOGIN_FAILURE: 'LOGIN_FAILURE',
-  LOGIN_SUCCESS: 'LOGIN_SUCCESS',
-  LOGOUT_FAILURE: 'LOGOUT_FAILURE'
+  FETCH_SUCCESS: 'user/FETCH_SUCCESS',
```

Comment:
Please use the new action type generator functions.

Reply:
Those actions will be replaced during a merge, cause the generator it on master right now